internal/sirius/add_feedback.go

```diff
@@ -0,0 +1,63 @@
+package sirius
+
+import (
+	"bytes"
+	"encoding/json"
+	"github.com/ministryofjustice/opg-sirius-user-management/internal/model"
+	"net/http"
+)
+
+func (c *Client) AddFeedback(ctx Context, form model.FeedbackForm) error {
+	var body bytes.Buffer
+	var err error
+
+	if len(form.Message) == 0 {
+		return ValidationError{
+			Message: "isEmpty",
+		}
+	}
+
+	if len(form.Message) > 900 {
+		return ValidationError{
+			Message: "stringLengthTooLong",
```

Comment:
Is there a test for this validation error?

Reply:
Yup TestAddFeedbackStringTooLongValidationError in add_feedback_test.go